### ISSUE TITLE
Transaction receipt gas used seems incorrect

### ISSUE THREAD

**Issue by k06a** [trufflesuite/ganache#977]
I got this tx receipt from `ganache-cli`.
```
const receipt = await contract.method();
```
```
receipt { transactionHash:
   '0x3747b3bf52faf7c048ff5e0945c92dd6309e0676eeddd703d42c478258f3039a',
  transactionIndex: 0,
  blockHash:
   '0xc6f3315a56029857f5db898b1cfab7443e7c90a2bdffbe65f4db8041156d45b9',
  blockNumber: 42,
  from: '0x9e1ef1ec212f5dffb41d35d9e5c14054f26c6560',
  to: '0x3d627fe11843ef6b3d5ec6683d53bd9822696ef6',
  gasUsed: '0x329c0e',
  cumulativeGasUsed: 91978,
  contractAddress: null,
  logs: []
}
```

I see no problem after fetching the same receipt again:
```
const receipt2 = await web3.eth.getTransactionReceipt(receipt.transactionHash);
```
```
  gasUsed: 91978,
  cumulativeGasUsed: 91978,
```

* I asked about this on [gitter](https://gitter.im/ConsenSys/truffle): [x]
* Link to my question on gitter: https://gitter.im/ConsenSys/truffle?at=5f1fed9fc1c0090e64aa447

## Expected Behavior

```
  gasUsed: 91978,
  cumulativeGasUsed: 91978,
```

## Current Behavior

```
  gasUsed: '0x329c0e',
  cumulativeGasUsed: 91978,
```

## Possible Solution

Seems like bug, this tx gas used is really 91k, not 3M of gas.

## Steps to Reproduce (for bugs)

Seems like should be reproducible for any tx receipt.

## Your Environment
* Version used: 6.9.1
* Version of Truffle/Remix/Other tools used: Truffle 5.1.35
* NodeJS Version: [ ] 6.x, [ ] 7.x (unsupported), [ ] 8.x, [ ] 9.x, [x] 10.x
* Operating System and version (include distro if Linux): macOS 10.15.4

**Comment by kevinbluer** [comment]
Hey @k06a! Apologies for the delay in getting to this one. If it's still an issue (or for anyone else experiencing something similar), it should be fixed in [ganache@7.0.0-alpha.0](https://github.com/trufflesuite/ganache/releases/tag/ganache%407.0.0-alpha.0).

Let us know how you get on 👍

**Comment by k06a** [comment]
@kevinbluer thx!
